policyengine_us/parameters/gov/states/ky/tax/income/deductions/standard.yaml

```diff
@@ -2,15 +2,19 @@ description: Kentucky provides this standard deduction amount.
 values:
   2021-01-01: 2_690
   2022-01-01: 2_770
+  2023-01-01: 2_980
 metadata:
+  uprating: gov.irs.uprating
```

Comment:
is this rounded? Need to check legal code, if so, we will need to add some extra uprating metadata:
e.g.
![Uploading image.png…]()

Reply:
please resolve this

Reply:
ok please leave a comment mentioning that